homework_20.py

```diff
@@ -0,0 +1,60 @@
+import random
+import requests
+
+
+#task 1
+
+
+websites = [
+    'http://google.com',
+    'http://facebook.com',
+    'http://twitter.com',
+    'http://amazon.com',
+    'http://apple.com'
+]
+
+random_website = random.choice(websites)
+
+response = requests.get(random_website)
+
+status_code = response.status_code
+website_name = random_website.split("//")[-1]
+html_length = len(response.text)
+
+print('Статус-код:', status_code)
+print('Назва сайту:', website_name)
+print('Довжина HTML-коду:', html_length)
+
+
+#task 2
+
+def get_city_coordinates(city):
+    geocoding_url = "https://api.open-meteo.com/v1/geocode"
+    params = {"query": city}
+    response = requests.get(geocoding_url, params=params)
+    data = response.json()
+
+    if "latitude" in data and "longitude" in data:
+        latitude = data["latitude"]
+        longitude = data["longitude"]
+        return latitude, longitude
+
+    return None
+def get_weather_data(latitude, longitude):
```

Comment:
nit: add two blank lines before and after function for more readability
```suggestion
    return None
    
    
def get_weather_data(latitude, longitude):
```